cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMakerCluster.java

```diff
@@ -0,0 +1,447 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.ContainerBuilder;
+import io.fabric8.kubernetes.api.model.ContainerPort;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.ServicePort;
+import io.fabric8.kubernetes.api.model.Volume;
+import io.fabric8.kubernetes.api.model.VolumeMount;
+import io.fabric8.kubernetes.api.model.extensions.Deployment;
+import io.fabric8.kubernetes.api.model.extensions.DeploymentStrategy;
+import io.fabric8.kubernetes.api.model.extensions.DeploymentStrategyBuilder;
+import io.fabric8.kubernetes.api.model.extensions.RollingUpdateDeploymentBuilder;
+import io.strimzi.api.kafka.model.CertAndKeySecretSource;
+import io.strimzi.api.kafka.model.CertSecretSource;
+import io.strimzi.api.kafka.model.KafkaMirrorMaker;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerAuthenticationScramSha512;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerAuthenticationTls;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerClientSpec;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerConsumerSpec;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerSpec;
+import io.strimzi.api.kafka.model.PasswordSecretSource;
+import io.strimzi.operator.common.model.Labels;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class KafkaMirrorMakerCluster extends AbstractModel {
+
+    private static final String NAME_SUFFIX = "-mirror-maker";
+    private static final String SERVICE_NAME_SUFFIX = NAME_SUFFIX + "-api";
```

Comment:
it should come from Kafka Connect, I don't think we need a suffix for the only service we have here for metrics, for sure the suffix should not be `-api` :-)